src/modules/grep/grepApi.tsx

```diff
@@ -13,12 +13,32 @@ import {
 } from '../../util/apiHelpers';
 import handleError from '../../util/handleError';
 
+interface Tekst {
+  tekst: {
+    spraak: string;
+    verdi: string;
+  }[];
+}
+
+interface Title {
+  spraak: string;
+  verdi: string;
+}
+
+const getTitles = async (titles: Tekst | Title[] | undefined) => {
```

Comment:
Ser ikkje heilt forskjellen på Tekst og Title[] i og med at Tekst er en array av objekter som er lik som Title.

Reply:
veldig bra suggestion der oppe ^  saken er at når man henter kompetansemål og kjerneelementer fra grep så er flyten slik tittel -> tekst -> (faktiske objektet), mens når man henter fra TT så er flyten tittel -> (faktiske objektet) altså de fra TT omitter et lag. [https://imgur.com/a/7W7BGls](LINK)

Reply:
Hvorfor kan ikke denne funksjonen uansett bare ta inn `Title[]` da?

Reply:
For jeg trenger å sjekke om title.tekst eksisterer altså for å teste om det er TT type eller KE/KM, og da kan jeg ekstrakte data (de ternary operators inn i getTitles() funksjonen) Med mindre du har andre forslag hvordan jeg kan sjekke det uten å bruke dobbel interface ?

Reply:
Og hvis jeg faktisk bruker bare Title så får jeg følgende error: [LINK](https://i.imgur.com/J8MLc7U.png)

Reply:
Jeg ville jo stemt for at du istedet for at du kaller denne med `x.tekst` der du har det ekstra laget.

Reply:
No work [LINK](https://i.imgur.com/vwhtEVm.png)

Reply:
Det var klønete av udir å ha forskjell på formatet på det som returneres.